content/en/core/overview/technical-roadmap.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Technical roadmap"
+linkTitle: "Technical roadmap"
+weight: 2
+description: >
+  Outlines the planned changes to technical aspects of the CHT.
```

Comment:
Would be good to keep punctuation and phrase style consistent with other pages. This isn't explicitly stated in the style guide, so I'll add it.

```suggestion
  Planned changes to technical aspects of the CHT
```